book/getting_started/system_requirements.md

```diff
@@ -0,0 +1,22 @@
+# System Requirements
+
+**Disclaimer**: The information provided is only a rough estimate based on current node implementations, and may not be representative of all node implementations. We plan to conduct benchmarking and optimization tests to ensure the best performance for each specific audience.
+
+## Hardware
+- 64-bit architecture.
+- We recommend a minimum of 16GB RAM 
+- We recommend using an SSD or NVMe, as an HDD may cause the node to fall behind the chain tip. Note that SSD performance can deteriorate when close to capacity.
+
+
+### Archive node
+- For an Archive node of Ethereum Mainnet, we recommend at least 3TB storage space: 1.8TB for state and 200GB for temp files 
+
+### Full node
+- For an Ethereum Mainnet Full node, we recommend at least 400GB of storage.
```

Comment:
On further thought let's make this TODO. We cannot recommend anything and don't want to mis set expectations. Thanks for the patience 😅